src/test/java/com/google/sps/AttendeeTest.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Date;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Attendee;
+
+@RunWith(JUnit4.class)
+
+public class AttendeeTest {
+
+  private final LocalServiceTestHelper helper =
+  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+   helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+   helper.tearDown();
+  }
+
+  @Test
+  public void attendeeTester() {
+    
+    // Creates test data. 
+    Attendee Test = new Attendee("12345", "Taniece", new Date());
```

Comment:
You could also create a new Date object separately, so you can test it as one of the get values